src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -1234,7 +1234,38 @@ class npc_scarlet_miner : public CreatureScript
         }
 };
 
-// npc 28912 quest 17217 boss 29001 mob 29007 go 191092
+class spell_stable_master_repo : public SpellScriptLoader
+{
+public:
+    spell_stable_master_repo() : SpellScriptLoader("spell_stable_master_repo") { }
+
+    class spell_stable_master_repo_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_stable_master_repo_AuraScript);
+
+        void HandleDummy(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
+        {
+            Creature* creature = GetTarget()->ToCreature();
+            if (!creature)
+                return;
+
+            if (Unit* owner = creature->GetCharmerOrOwner())
+                GetCaster()->GetAI()->AttackStart(owner);
```

Comment:
attack charmer? seems suspicious - pretty sure this will not work at this point since its friendly during the charm

Reply:
On which source the script is based?
Maybe this should be done here?
https://github.com/TrinityCore/TrinityCore/pull/20897/files#diff-baeb010bab3defc8b23a63d10965f7e8R829